docs/algorand/for-end-users/Algorand-Standard-Assets-support.mdx

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_position: 5
+---
+import asadispenser from "@site/static/img/asadispenser.png";
+import addasset from "@site/static/img/addasset.png";
+import EUReinwallet from "@site/static/img/EUReinwallet.png";
+import addingEuretowallet from "@site/static/img/addingEuretowallet.png";
+import EURewrappingchoice from "@site/static/img/EURewrappingchoice.png";
+import EURewrappingconfirmation from "@site/static/img/EURewrappingconfirmation.png";
+import ASAwrappedconfirmations from "@site/static/img/ASAwrappedconfirmations.png";
+import AddASSAtoMetamask from "@site/static/img/AddASSAtoMetamask.png";
+import ChooseASAtounwrap from "@site/static/img/ChooseASAtounwrap.png";
+import ConfirmASAunwrapping from "@site/static/img/ConfirmASAunwrapping.png";
+import tokenerc20allowance from "@site/static/img/tokenerc20allowance.png";
+import erc20allowance from "@site/static/img/erc20allowance.png";
+import unwrappingreqconfirmation from "@site/static/img/unwrappingreqconfirmation.png";
+
+
+# Algorand Standard Assets support 
+<p style={{textAlign:"justify"}}>
+This section gives brief introduction about Algorand Standard Assets (ASAs) supported by the Milkomeda A1 Rollup Bridge and how to move ASA (wrap or unwrap) between Milkomeda A1 Rollup and mainchain (Algorand).
+Before we will move to the step-by-step instruction it’s important to understand what ASAs are, please check the next part of this tutorial to get familiar with them.
+<h3>Algorand Standard Assets (ASAs)</h3>
+
+The Algorand Standard Assets (ASAs) are the on-chain assets supported by Algorand that ensures the same security, compatibility and speed as its native currency - Algo. 
+ASA can represent stablecoins, in-game currency, loyalty points etc. They can also represent single assets, like collectibles. 
+
+:::info
+ASAs that represent multiple assets within the same type can be called "fungible assets” - for example stablecoins. Those that are unique can be called non-fungible assets.<br />
```

Comment:
ASAs that are multiple assets of the same type can be called "fungible assets” - stablecoins, for example. Those that are unique are called non-fungible assets.<br />